### PR TITLE
Add documentation for operating condition

### PR DESCRIPTION
**Pull request by yifan-flex**

### CODE REVIEW

File: flow360/component/simulation/operating_condition/operating_condition.py [hunk 1]
```diff
@@ -342,17 +352,17 @@ def operating_condition_from_mach_reynolds(
...
     ----------
     mach : NonNegativeFloat, optional
-        The Mach number (must be non-negative). Default is None.
+        Freestream Mach number (must be non-negative).
     reynolds : PositiveFloat, optional
-        The Reynolds number (must be positive). Default is None.
+        Freestream Reynolds number defined with mesh unit (must be positive).
     temperature : TemperatureType.Positive, optional
-        The static temperature (must be a positive temperature value). Default is None.
+        Freestream static temperature (must be a positive temperature value). Default is ``288.15 * u.K``.
```
Comment (benflexcompute) [inline flow360/component/simulation/operating_condition/operating_condition.py:359]:
```suggestion
        Freestream static temperature (must be a positive temperature value). Default is ``288.15 Kelvin``.
```

File: flow360/component/simulation/operating_condition/operating_condition.py [hunk 2]
```diff
@@ -212,15 +212,24 @@ class AerospaceConditionCache(Flow360BaseModel):
 
 
 class AerospaceCondition(MultiConstructorBaseModel):
-    """Operating condition for aerospace applications."""
+    """
+    Operating condition for aerospace applications. Defines both reference paramenters used to compute nondimensional
```
Comment (benflexcompute) [inline flow360/component/simulation/operating_condition/operating_condition.py:216]:
```suggestion
    Operating condition for aerospace applications. Defines both reference parameters used to compute nondimensional
```

File: flow360/component/simulation/operating_condition/operating_condition.py [hunk 3]
```diff
@@ -212,15 +212,24 @@ class AerospaceConditionCache(Flow360BaseModel):
...
     velocity_magnitude: Optional[VelocityType.NonNegative] = ConditionalField(
-        description="Freestream velocity magnitude.", context=CASE
+        description="Freestream velocity magnitude. Used as reference velocity magnitude"
+        + " when :paramref:`reference_velocity_magnitude` is not specified.",
+        context=CASE,
+    )
+    thermal_state: ThermalState = pd.Field(
+        ThermalState(),
+        alias="atmosphere",
+        description="Reference and freestream thermal state. Defaults to US standard atomsphere at sea level.",
```
Comment (benflexcompute) [inline flow360/component/simulation/operating_condition/operating_condition.py:231]:
```suggestion
        description="Reference and freestream thermal state. Defaults to US standard atmosphere at sea level.",
```

File: flow360/component/simulation/operating_condition/operating_condition.py [hunk 4]
```diff
@@ -244,15 +253,16 @@ def from_mach(
...
-            The Mach number (non-negative).
+            Freestream Mach number (non-negative).
+            Used as reference Mach number when ``reference_mach`` is not specified.
         alpha : AngleType, optional
             The angle of attack. Defaults to ``0 * u.deg``.
         beta : AngleType, optional
             The side slip angle. Defaults to ``0 * u.deg``.
         thermal_state : ThermalState, optional
-            The atmospheric thermal state. Defaults to a standard :class:`ThermalState`.
+            Reference and freestream thermal state. Defaults to US standard atomsphere at sea level.
```
Comment (benflexcompute) [inline flow360/component/simulation/operating_condition/operating_condition.py:263]:
```suggestion
            Reference and freestream thermal state. Defaults to US standard atmosphere at sea level.
```

File: flow360/component/simulation/operating_condition/operating_condition.py [hunk 5]
```diff
@@ -342,17 +352,17 @@ def operating_condition_from_mach_reynolds(
     Parameters
     ----------
     mach : NonNegativeFloat, optional
-        The Mach number (must be non-negative). Default is None.
+        Freestream Mach number (must be non-negative).
     reynolds : PositiveFloat, optional
-        The Reynolds number (must be positive). Default is None.
+        Freestream Reynolds number defined with mesh unit (must be positive).
     temperature : TemperatureType.Positive, optional
```
[the remark this answers is not in the document]
Comment (yifan-flex) [reply flow360/component/simulation/operating_condition/operating_condition.py:358]: This one is optional. It has a default.

**Review by benflexcompute — APPROVED** [review]

**Pull request merged**